libdnf/dnf-goal.cpp

```diff
@@ -70,8 +69,19 @@ dnf_goal_depsolve(HyGoal goal, DnfGoalActions flags, GError **error)
         else
             g_string_append_printf(string, "%i problems detected:\n", cnt);
```

Comment:
What about supporting language translation?
Example of new code:
add #include "utils/bgettext/bgettext-lib.h"

and 
        string = g_string_new(_("Could not depsolve transaction; "));
        cnt = hy_goal_count_problems(goal);
        g_string_append_printf(string, P_("%i problem detected:\n", "%i problems detected:\n", cnt), cnt);
        for (j = 0; j < cnt; j++) {
            auto tmp = hy_goal_describe_problem_rules(goal, j);
            for (auto iter = tmp; *iter; ++iter) {
                if (tmp == iter) {
                    if (cnt != 1) {
                        g_string_append_printf(string, _(" Problem %1$i: %2$s\n"), j + 1, *iter);
                    } else {
                        g_string_append_printf(string, _(" Problem: %s\n"), *iter);
                    }